### PR TITLE
Add FreeBSD 14 CI

### PR DESCRIPTION
**Pull request by JohnTitor**
r? @ghost

### CODE REVIEW

**Comment by JohnTitor** [comment]
I'll r+ once https://github.com/rust-lang/homu/pull/160 is merged.

**Comment by JohnTitor** [comment]
Well, I guess we don't have to wait for the homu config change as it's still worth checking CI result w/o gating, @bors r+

**Comment by JohnTitor** [comment]
Hi, hi, hi, timeout!
@bors retry

File: src/unix/bsd/freebsdlike/freebsd/freebsd14/mod.rs [hunk 1]
```diff
@@ -0,0 +1,426 @@
...
+                    .f_mntonname
+                    .iter()
+                    .zip(other.f_mntonname.iter())
+                    .all(|(a,b)| a == b)
+            }
+        }
+        impl Eq for statfs {}
+        impl ::fmt::Debug for statfs {
+            fn fmt(&self, f: &mut ::fmt::Formatter) -> ::fmt::Result {
+                f.debug_struct("statfs")
```
Comment (Minoru) [inline src/unix/bsd/freebsdlike/freebsd/freebsd14/mod.rs:234]: This is missing `f_version`, `f_type` and `f_flags` — is that intentional?

**Pull request merged**